src/styles/theme.scss

```
@@ -27,6 +27,7 @@ $svgSizeMedium: 26px;
 
 /** Layout, spacing **/
 $defaultSpacing: 10px;
+$spacingXXSmall: 3px;
```

Stylelint (17.4.0)

[error] Expected variable name "spacingXXSmall" to be kebab-case (scss/dollar-variable-pattern)

(scss/dollar-variable-pattern)